### ISSUE TITLE
Use custom CNI for cluster create operation

### ISSUE THREAD

**Issue by jiayiwang7** [aws/eks-anywhere#6456]

**Comment by Cajga** [comment]
I am sorry to see that this did not make it to 0.21 but I am happy to see the new `Q42024` label on it. :)

Could you guys (@vivek-koppuru, @jiayiwang7, @d8660091, @csplinter) share with us if this feature will support to install the EKS-A cluster without kube-proxy?

From our side, having a few extra manual steps to replace the bundled Cilium with Cilium Enterprise or Community is not the real pain point. The real issue is that we cannot use Cilium's advanced features because we cannot [remove kube-proxy from EKS-A](https://github.com/aws/eks-anywhere/issues/6409).

**Comment by vivek-koppuru** [comment]
@Cajga Sorry about that but the labeling was a mistake. We don't have a plan for this immediately, but we have something tracking for exposing the cni-exclusive flag for Cilium. Does that help in this case or no?

**Comment by vivek-koppuru** [comment]
So is your main motivation for this issue to resolve https://github.com/aws/eks-anywhere/issues/6409?

**Comment by Cajga** [comment]
Correct! We hoped that the solution to this issue will include a fix to that issue as well. :/

A solution to that ticket would remove all of the remaining limitations of EKS-A for our current use cases.
